selectel/dbaas.go

```diff
@@ -138,3 +140,434 @@ func convertFieldToStringByType(field interface{}) string {
 func RandomWithPrefix(name string) string {
 	return fmt.Sprintf("%s_%d", name, rand.New(rand.NewSource(time.Now().UnixNano())).Int())
 }
+
+func flavorSchema() *schema.Resource {
+	return resourceDBaaSDatastoreV1().Schema["flavor"].Elem.(*schema.Resource)
+}
+
+func flavorHashSetFunc() schema.SchemaSetFunc {
+	return schema.HashResource(flavorSchema())
+}
+
+func convertFieldFromStringToType(fieldValue string) interface{} {
+	if val, err := strconv.Atoi(fieldValue); err == nil {
+		return val
+	} else if val, err := strconv.ParseFloat(fieldValue, 64); err == nil {
+		return val
+	} else if val, err := strconv.ParseFloat(fieldValue, 32); err == nil {
```

Comment:
Looks like this else-if block is useless because two lines above we parse this value with a larger bitSize arg.
If value can't be parsed with 64 bits size, then it won't be parsed with 32. And if it can be parsed with 64, then this attempt to parse with 32 won't be reached.

Reply:
Yeah, you are right, this part doesn't work correctly. Fixed that and moved to sdk